### DISCUSSION TITLE
Practice on custom sentence | New feature

### DISCUSSION THREAD

**Discussion by veerviren** [monkeytypegame/monkeytype#4170] [Ideas]
Recently, I have been thinking about sentences that contain **all the letters** of the alphabet, and I have found some that fulfill this requirement. I am eager to **practice** typing these sentences.

In my opinion, it would be helpful to add a **customer user-defined** paragraph feature for typing practice so that users can practice typing their favorite paragraphs and enjoy a customized typing experience. I would also like to contribute to adding this feature.

What are your thoughts on this idea? Please feel free to suggest.

**Comment by unrealapex** [comment]
Monkeytype already has a mode for practicing custom text:

![monkeytype custom text mode](https://user-images.githubusercontent.com/79121360/231260076-0805a19a-21b0-459f-a507-9a8deac5600b.png)

Quotes satisfy your request to practice their "favorite paragraphs".
